XcodeServerSDK/XcodeServer.swift

```diff
@@ -186,23 +186,55 @@ public extension XcodeServer {
             completion(success: false, error: Error.withInfo("Nil response"))
         }
     }
-
-    public func createBot(botOrder: Bot, completion: (bot: Bot?, error: NSError?) -> ()) {
-        
-        let body: NSDictionary = botOrder.dictionarify()
+    
+    public enum CreateBotResponse {
+        case Success(bot: Bot)
+        case BlueprintNeedsFixing(fixedBlueprint: SourceControlBlueprint)
+        case Error(error: ErrorType)
```

Comment:
Would we want to use `case Error` rather than `throws`?

I like receiving the error case but wouldn't it brake consistency with the rest of the code (as is being envisioned for `Swift 2.0`?)